_data/adobe-sid/adobe-sid.md

```diff
@@ -0,0 +1,7 @@
+---
+name: SIDDHANT MANGLIK # No longer than 18 characters
+institution: JAYPEE INSTITUTE OF INFORMATION TECHNOLOGY 🚩 # no longer than 58 characters
+profile_pic: sid.png # Name and extension of your profile picture(ex. mona.png)
```

Comment:
The name here does not match the image name that you have uploaded. I would suggest fixing this (make sure name is same in both the places) and as the other reviewer suggested - also update the checklist.

These things will make the moderators easy to review your PR.

Reply:
The image you have uploaded also does not follow the rules:
1) Should be 1:1 ratio (a square)
2) At least 544x544.

Please update your image also.

Reply:
thanks.
did it right